client/support/index.js

```diff
@@ -0,0 +1,36 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import {
+	storeUserAndToken,
+	getQueryParameter,
+	USER_PARAM,
+	TOKEN_PARAM,
+} from 'lib/user/support-user-interop';
+
+const cleanHistory = () => {
+	const url = window.location.toString();
+
+	if ( url.indexOf( '?' ) > 0 ) {
+		const cleanUrl = url.substring( 0, url.indexOf( '?' ) );
+		window.history.replaceState( {}, document.title, cleanUrl );
+	}
+};
+
+export default function() {
+	page( '/support-user', () => {
```

Comment:
I think we'll still end up needing to whitelist this with a systems request. But I think the approach here is solid!